clickhouse_driver/client.py

```diff
@@ -90,6 +90,12 @@ def __init__(self, *args, **kwargs):
         self.reset_last_query()
         super(Client, self).__init__()
 
+    def __enter__(self):
+        return self
```

Comment:
Could run a dummy query like `SELECT 1` here to test the server/client settings (check for unknown settings if `settings_is_important`)?